docs/notebooks/extreme_value_analysis.ipynb

```diff
@@ -22,7 +22,7 @@
   },
```

Comment:
<p>With my previous comment, I don't think that I'd keep this cell. Either way, this is not at the right place.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>